frontend/src/pages/list/List.pages.tsx

```diff
@@ -1,26 +1,28 @@
-import React, { useState, useEffect } from "react";
-import { COUNTRY } from "../../assets/data/Country.data";
+import React from "react";
+import { useRecoilState } from "recoil";
+import { dataState } from "../../atoms/data";
 import { City } from "../../components/city/City.component";
 import { MyDrawer } from "../../components/drawer/Drawer.component";
 import { MyText } from "../../components/myText/MyText.component";
 import { LANGUAGE } from "../../constants/language";
-// import { RegionService } from "../../services/regions.service";
+import fetchData from "../../scripts/alerts.json";
+
+const printByRegion = (name: any): any => {
+  return fetchData.filter((data) => data.region === name);
+};
 
 export const List = () => {
-  // const [data, setData] = useState([]);
-  // const [region, setRegion] = useState([]);
-  // const [loader, setLoader] = useState(false);
-  // useEffect(() => {
-  //   const fetchData = async () => {
-  //     const result = await RegionService.regions(`1`);
-  //     console.log(result);
-  //     const result = await RegionService.regions(`${regionId}`);
-  //     setLoader(true);
-  //     setRegion(await result.data);
-  //     await result;
-  //   };
-  //   fetchData();
-  // }, []);
+  const [d, setD] = useRecoilState(dataState);
```

Comment:
let's use explicit/descriptive names
```suggestion
  const [alerts, setAlerts] = useRecoilState(dataState);
```